eng/InstallRuntimes.proj

```diff
@@ -20,9 +26,14 @@
   -->
 
   <PropertyGroup>
-    <DailyTest Condition="'$(DailyTest)' == ''">false</DailyTest>
     <BuildArch Condition="'$(BuildArch)' == ''">$(Platform)</BuildArch>
     <BuildArch Condition="'$(BuildArch)' == ''">x64</BuildArch>
+    <PrivateBuildTesting>false</PrivateBuildTesting>
+    <PrivateBuildTesting Condition="'$(PrivateBuildPath)' != ''">true</PrivateBuildTesting>
+    <InternalReleaseTesting>false</InternalReleaseTesting>
+    <InternalReleaseTesting Condition="'$(DotnetRuntimeVersion)' != '' and '$(DotnetRuntimeVersion)' != 'default'">true</InternalReleaseTesting>
+    <DotnetRuntimeDownloadVersion Condition="'$(DotnetRuntimeDownloadVersion)' == '' or '$(DotnetRuntimeDownloadVersion)' == 'default'">$(DotnetRuntimeVersion)</DotnetRuntimeDownloadVersion>
```

Comment:
This should probably be 
```suggestion
    <DotnetRuntimeDownloadVersion Condition="'$(DotnetRuntimeDownloadVersion)' == ''>$(DotnetRuntimeVersion)</DotnetRuntimeDownloadVersion>
```
in case we leave only empty as a valid option (or default otherwise).

Reply:
Both pipeline variables are set to default currently so it needs to check for both.  Why would checking for default be a problem?

Reply:
Checking for the word default is not my issue, is having two things to mean the same thing. I should choose either empty or default.